packages/web3-eth-accounts/src/index.ts

```diff
@@ -15,6 +15,585 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+/**
+ * # web3.eth.accounts
+ * The web3.eth.accounts contains functions to generate Ethereum accounts and sign transactions and data.
+ *
+ * **_NOTE:_** This package has NOT been audited and might potentially be unsafe. Take precautions to clear memory properly, store the private keys safely, and test transaction receiving and sending functionality properly before using in production!
+ *
+ *
+ * To use this package standalone and use its methods use:
+ * ```ts
+ * import { create, decrypt } from 'web3-eth-accounts'; // ....
+ * ```
+ *
+ * To use this package within the web3 object use:
+ * ```ts
+ * import { Web3 } from 'web3';
+ *
+ * const web3 = new Web3(Web3.givenProvider || 'ws://some.local-or-remote.node:8546');
+ * // now you have access to the accounts class
+ * web3.eth.accounts.create();
+ * ```
+ *
+ * ## create
+ *
+ * Generates and returns a Web3Account object that includes the private and public key
+ *
+ * ### Parameters
+ * No parameters
+ * ### Returns
+ * Returns a {@link Web3Account} object that includes the private and public key
+ *
+ * ### Example
+ * ```ts
+ * web3.eth.accounts.create();
+ * {
+ * address: '0xbD504f977021b5E5DdccD8741A368b147B3B38bB',
+ * privateKey: '0x964ced1c69ad27a311c432fdc0d8211e987595f7eb34ab405a5f16bdc9563ec5',
+ * signTransaction: [Function: signTransaction],
+ * sign: [Function: sign],
+ * encrypt: [AsyncFunction: encrypt]
+ * }
+ * ```
+ *
+ * ## privateKeyToAccount
+ *
+ * Get an Account object from the privateKey
+ *
+ * ### Parameters
+ * privateKey -  string or buffer of 32 bytes
+ *
+ * ### Returns
+ * Returns a {@link Web3Account} object from the private key
+ *
+ * ### Example
+ * ```ts
+ * privateKeyToAccount("0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709");
+ * >    {
+ * 			address: '0xb8CE9ab6943e0eCED004cDe8e3bBed6568B2Fa01',
+ * 			privateKey: '0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
+ * 			sign,
+ * 			signTransaction,
+ * 			encrypt,
+ * 	}
+ *
+ * ```
+ *
+ * ## privateKeyToAddress
+ *
+ * Using a private key get the ethereum Address
+ *
+ * ### Parameters
+ * privateKey - string or buffer of 32 bytes
+ *
+ * ### Returns
+ *  Returns the Ethereum address
+ *
+ * ### Example
+ *
+ * privateKeyToAddress("0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709")
+ * >0xb8CE9ab6943e0eCED004cDe8e3bBed6568B2Fa01
+ *
+ * ## signTransaction
+ *
+ * Signs an Ethereum transaction with a given private key.
+ * ### Parameters
+ * transaction - Takes a transaction object, see {@link TxData} {@link AccessListEIP2930TxData} {@link FeeMarketEIP1559TxData}
```

Comment:
Linking for `TxData`, `AccessListEIP2930TxData` and `FeeMarketEIP1559TxData` doesn't work for me. Maybe because they are imported from `ethereumjs`?

Reply:
Yes I think tsdocs cannot refer to links that are dependancy types